src/drawing/DrawingPathGroup.tsx

```diff
@@ -13,31 +13,45 @@ export type Props = {
 
 export const DRAWING_BORDER_WIDTH = 1;
 
-export const DrawingPathGroup = ({
-    isActive = false,
-    pathGroup: {
-        paths,
-        stroke: { color, size },
-    },
-    rootEl,
-}: Props): JSX.Element => {
-    let strokeWidth = size;
+export const getStrokeWidths = (
+    initialWidth: number,
+    referenceEl: DrawingSVGRef | null,
+): { strokeWidth: number; strokeWidthWithBorder: number } | null => {
```

Comment:
Can we make this non-nullable and return `0`, assuming we can modify it to return only the stroke width?

Reply:
Will revert the changes to this file since I'll just use the `DrawingPathGroup` component in `DrawingCreator` instead